gradio/utils.py

```diff
@@ -290,6 +291,29 @@ def iter_py_files() -> Iterator[Path]:
         time.sleep(0.05)
 
 
+def gradio_deep_equal(a: Any, b: Any) -> bool:
```

Comment:
Suggest dropping the `gradio`:
```suggestion
def deep_equal(a: Any, b: Any) -> bool:
```